libbpf-rs/src/map.rs

```diff
@@ -69,19 +80,14 @@ impl OpenMap {
         if fd < 0 {
             return Err(Error::System(errno::errno()));
         }
-
-        let ret = unsafe { libbpf_sys::bpf_map__reuse_fd(self.ptr, fd) };
+        let reuse_result = self.reuse_fd(fd);
 
         // Always close `fd` regardless of if `bpf_map__reuse_fd` succeeded or failed
```

Comment:
nit: can you update this comment as well? It should be "regardless of if `reuse_fd` succeeded or failed"